src/app.ts

```diff
@@ -1,4 +1,45 @@
+import path from 'path';
+import logger from '../logger';
 import { User, Product } from 'models';
+import DirWatcher from 'utils/dirwatcher';
+import Importer from 'utils/importer';
+import csvConverter from 'utils/csvConverter';
+import { DIRECTORY_FILES_CHANGED } from 'constants/EventNames';
+
+const PATH_TO_DATA = path.join(__dirname, '../data');
+const WATCH_DELAY_MILLISECONDS = 2000;
```

Comment:
to config?

Reply:
For what reason? Who will manipulate with this config?